### PR TITLE
Muriel's mobile-friendly website

### PR DESCRIPTION
**Pull request by murielhoegger**
I struggled a lot with the circles, but I think I got it to work in the end.

### CODE REVIEW

**Comment by HIPPIEKICK** [comment]
Hi Muriel! You forgot to upload your folder called "pictures". Would it be possible for you to upload then and then write a comment and I will wait a bit with the code review? Would be nice to see your project in its entirety.

**Comment by murielhoegger** [comment]
Hi Matilda,

Oh no, so sorry!

Please find all the documents again in the file project-p2-layouts-master<https://github.com/murielhoegger/project-p2-layouts/tree/master/project-p2-layouts-master> (I uploaded everything again, but didn’t figure out how to delete)! I hope it works now.
Looking forward to your review…
BR, Muriel

On 11 Dec 2020, at 16:36, Matilda Arvidsson <notifications@github.com<mailto:notifications@github.com>> wrote:



Hi Muriel! You forgot to upload your folder called "pictures". Would it be possible for you to upload then and then write a comment and I will wait a bit with the code review? Would be nice to see your project in its entirety.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    <!-- dont forget the meta tag for responsiveness -->
+  </head>
+
+  <body>
+    <header>
+    <div class="hamburger">
+      <div></div>
+      <div></div>
+      <div></div>
+    </div>
```
Comment (HIPPIEKICK) [inline index.html:17]: Nice hamburger!

File: index.html [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+              Two of my biggest passions are <b> baking </b>and all things <b> Scandi</b>. <br> 
+              Here I share my recipes that let you bring a piece of Scandinavia in to your kitchen, <br> 
+              and I also link my favourite design pieces so that the rest of the house also gets that hygge feeling.
+            </p>
+          </div>
+        </section>
+      </section>
+
+    <section class="kitchen-container">
+      <section class="container">
```
Comment (HIPPIEKICK) [inline index.html:43]: Do you need both of these `section`s wrapping?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    <section class="interior-container">
+        <section class="container">
+          <h3>Scandinavian Interior Design Inspiration</h3>
+        </section>
+        <section class="interior">
+          <a href="#">
+            <div>
+              <p>Hay twisted candles</p>
+            </div>
+            <img class="interior-square-images" src="/pictures/hay-candles-coulours-square.jpg " alt="Hay Candles">
```
Comment (HIPPIEKICK) [inline index.html:101]: When an element has more than two attributes we can give them new lines for a better structure:
```
<img
  class="interior-square-images"
  src="/pictures/hay-candles-coulours-square.jpg"
  alt="Hay Candles">
```

File: index.html [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+              <p>IKEA Nilsove chair</p>
+            </div>
+            <img class="interior-square-images" src="/pictures/ikea-nilsove.jpeg" alt="IKEA Nilsove chair">
+          </a>
+        
+        </section>
+    </section>
+
+    <footer>
+      <div class="footer">
```
Comment (HIPPIEKICK) [inline index.html:122]: You can style the `footer` directly instead of putting a `div` inside it :)

File: style.css [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+  color: #fff;
+  text-decoration: none;
+  margin-left: 16px;
+}
+
+
+/*** WELCOME CONTAINER ***/
+
+.welcome-container .container { 
+  padding: 100px 16px;
```
Comment (HIPPIEKICK) [inline style.css:81]: This padding creates a horizontal scroll which we want to prevent when working with responsive design.

File: style.css [hunk 6]
```diff
@@ -0,0 +1,229 @@
...
+  width: 50%;
+  height:50%;
+}
+
+.interior a div {
+  border: 2px sold #d8eee6;
+  border-radius: 16px;
+}
+
+.interior-square-images {
```
Comment (HIPPIEKICK) [inline style.css:168]: Consider changing the class name as these are not square ;) Looking really good!

**Review by HIPPIEKICK — COMMENTED** [review]
Looking really good, thanks for sharing the pictures folder ⭐ Yummie pictures, clean design and great job with the flexboxing 💪 

Here are two games you could play to practice flexbox some more, they are actually quite fun 🤩
- [Flexbox Froggy](https://flexboxfroggy.com/)
- [Flexbox Zombies](https://flexboxzombies.com/p/flexbox-zombies)

**Pull request closed without merging**